### DISCUSSION TITLE
[已讀]功能讓人壓力好大

### DISCUSSION THREAD

**Discussion by Hsucuchia** [1101-GS4719-C/Python-Community-GS4719-C#1363] [General]
對於傳跟收回訊息的人來說已讀功能好像都很困擾
已讀功能從哪裡來的呢? Naver創辦人李海珍出差時遇上311大地震，無法及時聯絡家人，所以提出即時通訊軟體這個想法。已讀功能是為了救災，當災難發生時可以透過已讀功能迅速確認對方安全。然而現在卻變成了壓力的來源，不管是傳還是收訊息的人，其實都會有已讀的困擾。當傳訊息被已讀時，等回覆的人很緊張，剛開始可能會抱怨為什麼不回覆，但漸漸地也會猜想我是不是說錯話等等；收訊息的人也很焦慮，如果不方便回訊息時，不讀或已讀都會覺得很有負擔。因此彼此都要互相體諒!!

**Comment by ashleyy-huang** [comment]
我認為已讀的功能對於發送者是一種保障，它可以讓我確認到對方有收到我的訊息，不像是郵件，若我沒有收到回覆，我都不確定寄出的信件對方是否閱讀了，就算對方尚未回覆或已讀，這個功能可以讓我心裡有個底，至少對方有收到我的消息了

**Comment by 1104030360** [comment]
非常同意，我認為在現代社會已讀已經是許多交友關係中常常提起的詞彙，有些人甚至會因為被已讀焦慮一整天，以前的我也會因爲被已讀非常焦慮，但後來我體認到別人是因為有急事要處理不是故意的就不再那麼焦慮，我認為社群軟體可以推出相對應功能來解決這個問題，或是當我們有重要的是討論實盡量選擇實體方式討論！

**Comment by tower20021202** [comment]
我認為我們不應該過度去在意對方的已讀，而應該好好的保持自我，對方已讀代表他已經看到了解，過分解讀只會影響到自己，但我還是很討厭被已讀😂

**Comment by Alexgns** [comment]
我認為有時候別人可能忙於其他事情、無法立即回覆，或者只是需要更多時間來思考回應。保持耐心和理解對方的處境是很重要的。

**Comment by quan0715** [comment]
我覺得 按表情的功能非常實際，其實熟人之間在使用聊天軟體都往往是訊息的傳遞，無法實體傳達而已，有時候接收到訊息也未必要做出後續回應，的確，有時候已讀功能會讓人蠻焦慮的，但如果可以的話按個表情符號人與人之間的溫暖就會加倍。

**Comment by zephyrxin01** [comment]
我也覺得已讀功能雖然對某些人來說是種壓力，但仍然有它存在的必要性。尤其是討論的時候，如果沒有已讀的功能，有些人可能會藉機躺平假裝沒看到，發起討論的人也會滿焦慮大家的狀況。不過話說回來，大家有發現telegram可以長按聊天室，就能往前預覽所有對話紀錄嗎。這算是一個回訊息焦慮人可以避免已讀別人的好技巧。

**Comment by alicia0725** [comment]
我覺得已讀功能對不喜歡回訊息的人是個福音耶，透過已讀就能表達出我已經接收到了，不需要再特別進行回覆。至於有些人認為已讀一定要回覆的，預覽功能就也非常實際的可以運用上了

**Comment by pudding0803** [comment]
原來如此！

不過其實會因此焦慮的人也可以稍微想一下，當話題結束、意外按到、想先思考晚點再回覆等等，這些情況都是有可能的，任何人都滿有造成已讀的機會的，所以不用想太多 ~（雖然有時候我也會🥹）~

但因為其他原因故意已讀不回的又另當別論了🤧
